datasploit.py

```diff
@@ -45,45 +46,64 @@ def main(argv):
         shutil.copyfile(config_sample_path,config_file_path)
         print "[+] A config file is added please follow guide at https://datasploit.github.io/datasploit/apiGeneration/ to fill API Keys for better results"
         # We can think about quiting at this point.
-    # if no argument is provided print help and quit
-    if len(argv) == 0:
-        parser.print_help()
-        sys.exit()
     # parse arguments in case they are provided.
     x=parser.parse_args()
     active=x.active
     quiet=x.quiet
-    user_input=x.target
+    single_input=x.single_target
+    file_input=x.file_target
     output=x.output
+    # if no target is provided print help and quit.
+    if not (single_input or file_input):
+        print "\nSingle target or file input requered to run\n"
```

Comment:
spelling "required"

Reply:
Thanks for this catch! I'll get a new commit fixing this pushed shortly.